model/risk_model.py

```
@@ -1,7 +1,810 @@
+1. 우선순위 기반 위험도 판정 (SAFE/WARN/DANGER)
+2. 행정안전부 API로 정확한 주소 검증
+3. 4개 카테고리별 상세 분석 내용 생성
+4. Spring DetailGroup 형태로 정확한 결과 반환
+"""
+
+import sys
+import os
+import logging
+import re
+import requests
+from typing import Dict, List, Any, Optional
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)